sbt_common/display/src/test/scala/uk/ac/wellcome/display/test/util/JsonMapperTestUtil.scala

```diff
@@ -3,9 +3,9 @@ package uk.ac.wellcome.display.test.util
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.google.inject.{Guice, Injector}
 import uk.ac.wellcome.display.modules.DisplayJacksonModule
-import uk.ac.wellcome.test.utils.JsonTestUtil
+import uk.ac.wellcome.json.utils.JsonAssertions
 
-trait JsonMapperTestUtil extends JsonTestUtil {
+trait JsonMapperTestUtil extends JsonAssertions {
 
```

Comment:
`JsonAssertions` is nicer

Reply:
Looks fine to me, but I think @alicefuzier should take a look because Json parsing is so critical. 

I'm not sure why we have introduced `JsonDecodingError`.